src/nodes.coffee

```diff
@@ -1498,11 +1510,14 @@ exports.Access = class Access extends Base
     else
       [@makeCode('['), name..., @makeCode(']')]
 
+  shouldCache: NO
```

Comment:
I was thinking that a good general pattern would be something like:

- Initialization methods: constructor, etc.
- Miscellaneous methods: getters, helpers, etc.
- Output methods: `compileNode`, `ast`, etc.

From skimming through the code it looks maybe 50-50 between helpers in the middle and at the end. I think what happened was that the pattern was supposed to be what I’m proposing here, but as various contributors have added functionality over the years the new methods tended to get added at the end of each class. If it’s okay with you I’d like to try to at least keep our AST methods at the end, and we could do a quick cleanup PR after the AST stuff is merged in to move all the `compile*` methods just before the AST methods? Or alternatively, when the `compile*` methods go away as part of adding the printer, we’re already following the intended pattern since the AST methods are already at the end.